source/manual/incorrect-postcode-data.html.md

```diff
@@ -6,16 +6,34 @@ layout: manual_layout
 section: 2nd line
 ---
 
-We use postcode data from the [ONS Postcode Directory](https://geoportal.statistics.gov.uk/search?q=ONS%20Postcode%20Directory%20) (ONSPD) and Ordnance Survey for boundary
-line data. We typically [import the data into Mapit](https://github.com/alphagov/mapit/blob/master/IMPORTING-DATA.md) every 6 months.
+We get our postcode data from Locations API, which uses OS Places API (Ordnance Survey) under the hood. Locations API postcodes [keep themselves up to date automatically](https://github.com/alphagov/locations-api/blob/main/docs/postcodes-added-cached-updated.md#how-postcodes-are-updated) and should never be more than about a week out of date.
 
-We sometimes receive Zendesk tickets about postcodes not returning the right results.
-You can check the accuracy of the postcode by:
+In case the data is incorrect, a first step would be checking the Locations API result with the OS Places API result.
 
-* Looking up the postcode on Mapit, for example https://mapit.mysociety.org/postcode/E18QS
-  and viewing the parents. There can be more than one parent which can be the case
-  if the postcode sits between two boundaries
-* Get the latitude/longitude and look it up on Google Maps
+```shell
```

Comment:
I'd split this into two code blocks, one for `shell` and one for `ruby`, to get the correct highlighting in both.

Reply:
Yes, good point. Will do this now.